### DISCUSSION TITLE
Can we cancel a google Subscription using LIAP

### DISCUSSION THREAD

**Discussion by ghost** [imdhemy/laravel-in-app-purchases#384] [Q&A]
I'm trying to cancel a subscription from the server side and I have implemented but using google/apiclient library, I'd like to know if is possible to do using this amazing library LIAP?

**Comment by imdhemy** [comment]
Hi @vincy-dev93 

You need to upgrade to liap v`1.10.0` then you can use the `cancel()` method from the `Subscription` Facade as follows:

```php
use Imdhemy\Purchases\Facades\Subscription;

Subscription::googlePlay()->id($itemId)->token($purchaseToken)->cancel();
```

**Reply by ghost** [reply]
Thanks for that, is amazing the way you maintain this library @imdhemy, you're doing an amazing work on it..
